tests/security/test_secret.py

```diff
@@ -118,21 +116,15 @@ def test_token_inputs(db, config, encrypt, default_account):
     secret_id = default_account.refresh_token_id
     secret = db.session.query(Secret).get(secret_id)
 
-    assert not isinstance(
-        secret.secret, builtins.unicode if sys.version_info < (3,) else str
-    ), "secret cannot be unicode"
+    assert isinstance(secret.secret, bytes), "secret cannot be unicode"
     assert secret.secret == unicode_token.encode(
         "utf-8"
     ), "token not decrypted correctly"
 
-    with pytest.raises(ValueError) as e:
+    with pytest.raises(ValueError):
         default_account.refresh_token = invalid_token
 
-    assert e.typename == "ValueError", "token cannot be invalid UTF-8"
```

Comment:
This was not testing anything because it's already asserted that it's ValueError in pytest.raises